EpikCord/guild.py

```diff
@@ -534,8 +532,8 @@ def __init__(self, data: discord_typings.WelcomeScreenData):
 
 class GuildWidgetSettings:
     def __init__(self, data: dict):
-        self.enabled: bool = data.get("enabled")
-        self.channel_id: Optional[str] = data.get("channel_id")
+        self.enabled: bool = data["enabled"]
+        self.channel_id: Optional[int] = int(data.get("channel_id"))
```

Comment:
```py
>>> int(None)
....
TypeError: int() argument must be a string, bytes-like object or a number, not "NoneType"
```
If the field always exists, then might aswell use `data[...]` instead of `data.get(...)`